diff --git a/config.def.h b/config.def.h
index 9e61010..fd97bd0 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char font[] = "Consolas:pixelsize=18:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
